src/test/java/com/cloudbees/jenkins/plugins/kubernetes_credentials_provider/convertors/AWSCredentialsConvertorTest.java

```diff
@@ -185,33 +185,6 @@ public void canConvertAValidScopedSecret() throws Exception {
         }
     }
 
-    @Test
```

Comment:
please do not reduce the test coverage.  if it is valid to have an AWS secret without an access key or secret key then these tests need to be adjusted for the positive case and assert the returned credential is as expected.